builtin/providers/aws/cloudfront_distribution_configuration_structure_test.go

```diff
@@ -337,6 +363,9 @@ func TestCloudFrontStructure_flattenCacheBehavior(t *testing.T) {
 	if out["target_origin_id"] != "myS3Origin" {
 		t.Fatalf("Expected out[target_origin_id] to be myS3Origin, got %v", out["target_origin_id"])
 	}
+	if reflect.DeepEqual(out["lambda_function_associations"], in["lambda_function_associations"]) != true {
```

Comment:
I have a feeling this is passing because `reflect.DeepEqual` is comparing `nil` to `nil`. The singular version is used in [defaultCacheBehaviorConf](https://github.com/hashicorp/terraform/pull/11291/files#diff-58242079a8a0215c803ef662dacf67e4R19), so the plural `lambda_function_associations` doesn't exist.